tasks/elasticsearch-Debian.yml

```diff
@@ -14,7 +14,7 @@
 
 - name: Debian - Add Elasticsearch repository key
   become: yes
-  apt_key: url="{{ es_apt_key }}" state=present
+  apt_key: url="{{ es_apt_key }}" id=46095ACC8548582C1A2699A9D27D666CD88E42B4 state=present
```

Comment:
Can you make this a variable please? Something like `es_apt_key_id: 46095ACC8548582C1A2699A9D27D666CD88E42B4` to make sure that it can be overriden for anyone who is overriding the `es_apt_key`.

Reply:
I just merged upstream into my branch and modified my changes accordingly. The Key-ID of the ES-APT-key now is configurable with the default set in var/Debian.yml